src/connection.ts

```diff
@@ -779,6 +779,14 @@ export interface ConnectionOptions {
    */
   rowCollectionOnRequestCompletion?: boolean;
 
+  /**
+   * A string, that will allow user to provide their custom Common Name (CN) which matches a Common Name (CN)
+   * present in the server certificate.
+   *
+   * (no default)
+   */
+  serverName?: string;
```

Comment:
Small nit: Use `servername` instead, that would match the option name used by Node.js.